code/API_definitions/qod-api.yaml

```diff
@@ -40,8 +40,35 @@ paths:
         required: true
       callbacks:
         notifications:
-          "{$request.body#/notificationUrl}/notifications":
-            $ref: "#/paths/~1notifications"
+          "{$request.body#/webhook/notificationUrl}":
```

Comment:
I think we have to specify the security here for this operation, to be [Bearer Authentication](https://swagger.io/docs/specification/authentication/bearer-authentication/), as dictated by guidelines:

```
security:
  - notificationsBearerAuth: []
```

And list it under securitySchemes

```
components:
  securitySchemes:
    notificationsBearerAuth:
      type: http
      scheme: bearer
      bearerFormat: {$request.body#/webhook/notificationAuthToken}
```

`bearerFormat` value is only specified as a string (for documentation purposes), but using this notation may be clarifying

Reply:
@jlurien I can add it